### PR TITLE
Add cluster_tools recipe

### PR DESCRIPTION
**Pull request by constantinpape**
Add cluster tools, a python library for distributed bioimage analysis

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by constantinpape** [comment]
@conda-forge/help-python this PR is ready for review. Thanks!

File: recipes/cluster_tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set version = "0.3.5" %}
+{% set sha256 = "52eabb9474320f44a8c71d282c2c2a1bff20134d3f177a1b794d37a9f5b6f931" %}
```
Comment (chrisburr) [inline recipes/cluster_tools/meta.yaml:2]:
```suggestion
```

File: recipes/cluster_tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+{% set name = "cluster_tools" %}
+
+package:
+  name: cluster_tools
+  version: {{ version }}
+
+source:
+  url: https://github.com/constantinpape/cluster_tools/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (chrisburr) [inline recipes/cluster_tools/meta.yaml:11]:
```suggestion
  sha256: 52eabb9474320f44a8c71d282c2c2a1bff20134d3f177a1b794d37a9f5b6f931
```

**Comment by constantinpape** [comment]
Thanks for the review @chrisburr! Any other changes necessary before this can be merged?

**Comment by constantinpape** [comment]
Thanks @chrisburr!

**Pull request merged**